### PR TITLE
AI Generative article

### PR DESCRIPTION
**Pull request by bpascazio**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/ai/concepts/generative-ai/generative-ai.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+In some sense the change is expected to be welcome as jobs that are currently mundane or have been difficult to automate will go away. For example frame editing, common artwork generation, and basic copywriting.
+
+One concern with this reduction is how jobs typically mentor juniors. If the mundane entry work goes away - how will the next generation be trained?
+
+### Possibilities
+
+While any new disruptive technology creates a lot of uncertainties many people are also excited about what the future may bring.
+
+Generative AI may open up entirely new ways we can create and consume content. For example media now is difficult and time consuming to create even just in one length and format. Generative AI may allow different versions of movies, stories, and short videos to be generated from the same source media to allow new ways of media consumption.
```
Comment (HishamT) [inline content/ai/concepts/generative-ai/generative-ai.md:164]:
```suggestion
Generative AI may open up entirely new ways to create and consume content. For example, currently, any form of media is difficult and time-consuming to create. Generative AI may allow different versions of movies, stories, and short videos to be generated from the same source to allow new ways of media consumption.
```

File: content/ai/concepts/generative-ai/generative-ai.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+Generation of speech based on text with high fidelity style will allow creates to make content where they were limited by voice overlay before.
```
Comment (HishamT) [inline content/ai/concepts/generative-ai/generative-ai.md:166]:
```suggestion
The generation of high-fidelity speech based on text will allow creators to make content when previously, they were limited by voice overlay.
```

File: content/ai/concepts/generative-ai/generative-ai.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+Code generation, while automating repetitive work may result in higher quality code and the focus on new creative user centric solutions.
```
Comment (HishamT) [inline content/ai/concepts/generative-ai/generative-ai.md:168]:
```suggestion
Code generation, while automating repetitive work, may result in higher-quality code and allow engineers to focus on creative user-centric solutions.
```

File: content/ai/concepts/generative-ai/generative-ai.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+Movies and video creation once expensive and available through large studios and digital production groups can instead be created by individuals and bound only by their imagination.
```
Comment (HishamT) [inline content/ai/concepts/generative-ai/generative-ai.md:170]:
```suggestion
Movies and video creation, once expensive and available only through large studios and digital production groups, can instead be created by individuals and bound only by their imagination.
```

File: content/ai/concepts/generative-ai/generative-ai.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+Content may be connected back through the block chain to compensate original creators.
+
+Generative AI may help people understand more how creative processes work in general and how to continually enhance the human process.
```
Comment (HishamT) [inline content/ai/concepts/generative-ai/generative-ai.md:174]:
```suggestion
Generative AI can help people further understand how the creative process works in general and how to continually enhance the human process.
```

Comment (HishamT) [inline content/ai/concepts/generative-ai/generative-ai.md:174]: "the human process" is unclear. Can a different example be used?

**Review by HishamT — CHANGES_REQUESTED** [review]
Hey @bpascazio, thank you for your entry. I have reviewed it and it does appear to contain several problems.

1. Bold terms should be reserved for technical terms only.
2. There are a number of sentences that are unclear and should be rewritten. I have marked them.

Thank you again.

**Comment by SSwiniarski** [comment]
Closing this as it is replaced by [2043](https://github.com/Codecademy/docs/pull/2043).

**Pull request closed without merging**